plugins/scaffolder/src/next/TemplateWizardPage/TemplateWizardPage.tsx

```diff
@@ -73,6 +77,22 @@ export const TemplateWizardPage = (props: TemplateWizardPageProps) => {
     name: templateName,
   });
 
+  const [editUrl, setEditURL] = React.useState('');
+  catalogApi.getEntityByRef(templateRef).then(data => {
+    const templateEditUrl =
+      data?.metadata.annotations?.['backstage.io/edit-url'];
+    if (templateEditUrl !== undefined) setEditURL(templateEditUrl);
+  });
```

Comment:
```suggestion
const { value: editUrl } = useAsync(async () => {
  const data = await catalogApi.getEntityByRef(templateRef);
  return data?.metadata.annotations?.['backstage.io/edit-url'] || '';
}, [templateRef, catalogApi]);
```

Reply:
thank you.
i merged your suggestion
https://github.com/backstage/backstage/pull/23363/commits/96511604f64e5e0d5cb34dc790ab7117f6a57474